SPECS/dnf-plugins-core/dnf-plugins-core.spec

```diff
@@ -0,0 +1,966 @@
+ln -sf %{_libexecdir}/dnf-utils %{buildroot}%{_bindir}/yum-debug-dump
+ln -sf %{_libexecdir}/dnf-utils %{buildroot}%{_bindir}/yum-debug-restore
+ln -sf %{_libexecdir}/dnf-utils %{buildroot}%{_bindir}/yum-groups-manager
+ln -sf %{_libexecdir}/dnf-utils %{buildroot}%{_bindir}/yumdownloader
+
+%files
+%license COPYING
+%doc AUTHORS README.rst
+%{_mandir}/man8/dnf-builddep.*
+%{_mandir}/man8/dnf-changelog.*
+%{_mandir}/man8/dnf-config-manager.*
+%{_mandir}/man8/dnf-copr.*
+%{_mandir}/man8/dnf-debug.*
+%{_mandir}/man8/dnf-debuginfo-install.*
+%{_mandir}/man8/dnf-download.*
+%{_mandir}/man8/dnf-generate_completion_cache.*
+%{_mandir}/man8/dnf-groups-manager.*
+%{_mandir}/man8/dnf-needs-restarting.*
+%{_mandir}/man8/dnf-repoclosure.*
+%{_mandir}/man8/dnf-repodiff.*
+%{_mandir}/man8/dnf-repograph.*
+%{_mandir}/man8/dnf-repomanage.*
+%{_mandir}/man8/dnf-reposync.*
+%{_mandir}/man1/yum-changelog.*
+%{_mandir}/man8/yum-copr.*
+
+%exclude /usr/local/libexec/dnf-utils-2
+%exclude %_libdir/python2.7/site-packages/dnf-plugins/*
+%exclude %_libdir/python2.7/site-packages/dnfpluginscore/
+%exclude %{_mandir}/man1/yum-changelog.1.gz
+%exclude %{_mandir}/man8/*.8.gz
+%exclude %{_mandir}/man5/yum-versionlock.conf.5.gz
+%exclude /usr/local/share/man/*
+%exclude /usr/local/share/locale/*/LC_MESSAGES/dnf-plugins-core.mo
+%exclude %{_datadir}/locale/*/LC_MESSAGES/dnf-plugins-core.mo
+%exclude %_sysconfdir/dnf/plugins/local.conf
+%exclude %_sysconfdir/dnf/plugins/post-transaction-actions.conf
+
+%files -n python3-%{name}
+%{python3_sitelib}/dnf-plugins/*
+%{python3_sitelib}/dnfpluginscore/
+%_sysconfdir/dnf/plugins/copr.conf
+%_sysconfdir/dnf/plugins/debuginfo-install.conf
+%_sysconfdir/dnf/plugins/versionlock.list
+%_sysconfdir/dnf/plugins/versionlock.conf
+
+%files -n dnf-utils
+%{_libexecdir}/dnf-utils
+%{_bindir}/debuginfo-install
+%{_bindir}/needs-restarting
+%{_bindir}/find-repos-of-install
+%{_bindir}/package-cleanup
+%{_bindir}/repo-graph
+%{_bindir}/repoclosure
+%{_bindir}/repodiff
+%{_bindir}/repomanage
+%{_bindir}/repoquery
+%{_bindir}/reposync
+%{_bindir}/repotrack
+%{_bindir}/yum-builddep
+%{_bindir}/yum-config-manager
+%{_bindir}/yum-debug-dump
+%{_bindir}/yum-debug-restore
+%{_bindir}/yum-groups-manager
+%{_bindir}/yumdownloader
+%{_mandir}/man1/debuginfo-install.*
+%{_mandir}/man1/needs-restarting.*
+%{_mandir}/man1/repo-graph.*
+%{_mandir}/man1/repoclosure.*
+%{_mandir}/man1/repodiff.*
+%{_mandir}/man1/repomanage.*
+%{_mandir}/man1/reposync.*
+%{_mandir}/man1/yum-builddep.*
+%{_mandir}/man1/yum-config-manager.*
+%{_mandir}/man1/yum-debug-dump.*
+%{_mandir}/man1/yum-debug-restore.*
+%{_mandir}/man1/yum-groups-manager.*
+%{_mandir}/man1/yumdownloader.*
+%{_mandir}/man1/package-cleanup.*
+%{_mandir}/man1/dnf-utils.*
+%{_mandir}/man1/yum-utils.*
+
+%changelog
+* Fri Jul 30 2021 Neha Agarwal <nehaagarwal@microsoft.com> - 4.0.22-2
+- Initial CBL-Mariner import from rpm-software-management (license: GPLv2+)
```

Comment:
Double check the license of the source code and add a License Verified.

Reply:
Done